Cargo.toml

```diff
@@ -10,15 +10,15 @@ edition = "2018"
 
 [dependencies]
 auto_impl = "1.0"
-ethereum = { git = "https://github.com/rust-ethereum/ethereum.git", rev = "3be0d8fd4c2ad1ba216b69ef65b9382612efc8ba", default-features = false }
+ethereum = { git = "https://github.com/rust-ethereum/ethereum.git", rev = "bbb544622208ef6e9890a2dbc224248f6dd13318", default-features = false }
```

Comment:
@sorpaas could we have a new release for this package?

Reply:
Yes, it seems fine.

cc: @sorpaas @boundless-forest @koushiro 
If you guys agree, we should be able to merge this.